config/config.php

```diff
@@ -35,7 +35,12 @@
  * "Powered by SugarCRM".
  ********************************************************************************/
 
-$storageType = 'mysql';
+if (!empty($opts['storage'])) {
+    $storageType = $opts['storage'];
+} else {
+    $storageType = 'mysql';
+}
+
```

Comment:
IMHO it would be better to leave this file as is and add `else` clause [here](https://github.com/sugarcrm/Tidbit/blob/30f575942a00b02417e5d5c7a0a7d69040720640/install_cli.php#L94). This way we have a single place where the storage type is being determined.